experimental/generation/generator/packages/library/templates/generator.lg

```diff
@@ -558,18 +542,30 @@ ${if(length(entities[entity]) > 1, `    - @ ${entity} ${entity}`, '')}```
 
 > Name for property entity
 # propertyEntity()
-- ${`${property}Property`}
+- ${`${templateName(property)}Property`}
 
-> Check to see if entity is only found in one property
 # oneProperty(entity)
+> Check to see if entity is only found in one property
 - ${count(entities[entity]) == 1}
 
-> Test to see if property is an array
 # isArrayProperty(property)
+> Test to see if property is an array
 - ${propertySchema.type == 'array'}
 
-> Generate a raw entity value with optional label
+# isUnitsEntity(entity)
+> Entities that have units
+- ${contains(['age', 'dimension', 'money', 'temperature'], entity)}
+
+# units
+> Property units if specified
+- ${propertySchema.$units}
+
+# isLUISEntity(entity)
+> Assume LUIS entity unless built-in utterance
+- ${entity != 'utterance'}
```

Comment:
I think BreadEntity is also not built-in LUIS entity

Reply:
But it is defined in LUIS as a custom entity.

---
In reply to: [570818149](https://github.com/microsoft/BotBuilder-Samples/pull/3061#discussion_r570818149) [](ancestors = 570818149)